compose/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+import org.jetbrains.kotlin.gradle.dsl.kotlinExtension
+
+plugins {
+    alias(libs.plugins.androidGradlePlugin)
+    alias(libs.plugins.kotlin)
+    `maven-publish`
+}
+
+kotlinExtension.jvmToolchain(19)
+
+android {
+    namespace = "com.ioki.textref"
+    compileSdk = 33
+    defaultConfig {
+        minSdk = 21
+    }
+    buildFeatures.compose = true
+    composeOptions.kotlinCompilerExtensionVersion = libs.tools.android.compose.get().version
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    kotlinOptions { jvmTarget = "1.8" }
+}
+
+dependencies {
+    // Implementation
+    api(project(":core"))
+    api(libs.bundles.android.compose)
+}
+
+android.publishing {
+    singleVariant("release") {
+        withSourcesJar()
+        withJavadocJar()
+    }
+}
+
+val projectVersion = version as String
+publishing {
+    publications {
+        register<MavenPublication>("release") {
+            groupId = "com.github.ioki-mobility"
+            artifactId = "TextRef-Compose"
```

Comment:
Are these the right ID:s? They don't match up with the instructions in the readme 🤔

Reply:
What does "right" mean? 🤔 😁 
Jitpack works differently and will ignore the `groupId` and `artifactId` defined here anyways.
They are only used for native publishing...

Nevertheless, I think you are right and I should adjust it 👍